app/(auth)/login/LoginForm.tsx

```diff
@@ -32,10 +32,11 @@ export default function LoginForm(props: { returnTo?: string | string[] }) {
 
         if (returnTo) {
           router.push(returnTo);
+          router.refresh();
```

Comment:
what was this for again? maybe we should add comments above each `router.refresh()` ?

Reply:
https://nextjs.org/docs/app/api-reference/functions/use-router#userouter